cliquet/permission/redis.py

```diff
@@ -79,6 +79,23 @@ def object_permission_principals(self, object_id, permission):
         return self._decode_set(members)
 
     @wrap_redis_error
+    def principals_accessible_objects(self, principals, permission,
```

Comment:
Actually we don't need to because we already checked the other permissions before.